packages/amplify-graphql-transformer-core/src/utils/schema-utils.ts

```diff
@@ -1,24 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
 import { DynamoDbDataSource } from '@aws-cdk/aws-appsync';
 import { Table } from '@aws-cdk/aws-dynamodb';
 import * as assert from 'assert';
-import { ObjectTypeDefinitionNode } from 'graphql';
+import { ListValueNode, ObjectTypeDefinitionNode, StringValueNode } from 'graphql';
 import { ModelResourceIDs } from 'graphql-transformer-common';
 
-export function getKeySchema(table: any, indexName?: string): any {
-  return (
-    (
-      table.globalSecondaryIndexes.find((gsi: any) => gsi.indexName === indexName) ??
-      table.localSecondaryIndexes.find((gsi: any) => gsi.indexName === indexName)
-    )?.keySchema ?? table.keySchema
-  );
-}
+/**
+ * getKeySchema
+ */
+export const getKeySchema = (table: any, indexName?: string): any => (
+  (
+    table.globalSecondaryIndexes.find((gsi: any) => gsi.indexName === indexName)
+      ?? table.localSecondaryIndexes.find((gsi: any) => gsi.indexName === indexName)
+  )?.keySchema ?? table.keySchema
+);
 
-export function getTable(ctx: TransformerContextProvider, object: ObjectTypeDefinitionNode): any {
+/**
+ * getTable
+ */
+export const getTable = (ctx: TransformerContextProvider, object: ObjectTypeDefinitionNode): any => {
   const ddbDataSource = ctx.dataSources.get(object) as DynamoDbDataSource;
   const tableName = ModelResourceIDs.ModelTableResourceID(object.name.value);
   const table = ddbDataSource.ds.stack.node.findChild(tableName) as Table;
 
   assert.ok(table);
   return table;
-}
+};
+
+/**
+ * getSortKeyFieldNames
+ */
+export const getSortKeyFieldNames = (type: ObjectTypeDefinitionNode): string[] => {
```

Comment:
Is this function here as opposed to `graphql-index-transformer` so we can access this in the other resolvers without introducing a dependency between packages?